client/my-sites/checkout/composite-checkout/composite-checkout.tsx

```diff
@@ -311,7 +305,7 @@ export default function CompositeCheckout( {
 	);
 
 	const { storedCards, isLoading: isLoadingStoredCards, error: storedCardsError } = useStoredCards(
-		getStoredCards || wpcomGetStoredCards,
+		wpcomGetStoredCards,
```

Comment:
❤️

Reply:
Yeah, from what I can tell that prop isn't in use anywhere. Presumably it was there for testing at some point?